build.gradle

```diff
@@ -55,6 +56,28 @@ clean {
 	}
 }
 
+buildInDocker {
+	LinkedList commands = new LinkedList <String> ();
+
+	commands.addAll(['docker', 'build', '--tag', 'liferay-gradle', '.'])
```

Comment:
can we split this into two tasks?  one for building the image and another for running the container?

Reply:
Also we should give a tag that is `liferay-blade-cli-build`